deploy/olm-catalog/camel-k-dev/1.4.0/camel.apache.org_integrations.yaml

```diff
@@ -88,6 +88,21 @@ spec:
                 items:
                   type: string
                 type: array
+              errorHandler:
```

Comment:
Does it needs to be a top level property on the integration? I see there's a source of type "errorHandler" that can be added.

Also, you're updating the OLM manifest for 1.4.0 which is already released (I plan to only keep latest from now on btw, so it's not a real issue)

Reply:
After reading the PR, this seems something from previous attempts...

Reply:
That's correct, it sneaked in due to a change I did in the `Integration` in an interim commit. The autogen did not clean it up probably, so, removing that manually.